modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java

```diff
@@ -21,11 +21,13 @@
  */
 public class KubernetesPovisioningConstants {
 
-    public static final String KUB_MASTER_URL = "https://192.168.19.249/";
-    public static final String KUB_API_VERSION = "v1";
+    public static final String KUB_MASTER_URL = "KubernetesClusterConfig.{0}.Property.MasterURL";
```

Comment:
In here constant "KUB_MASTER_URL" does not hold the actual Kubernetes master url. It represents the configuration/property to look up, to get the url. Therefore I believe it is better if it is changed to something like PROPERTY_KUB_MASTER_URL such that the constant name will reflect, what it actually holds.
Same applies for below 3 constants as well.